gcn_email/core.py

```diff
@@ -27,10 +28,32 @@
 log = logging.getLogger(__name__)
 
 
+REPLACEMENT_TEXT = (
+    "This content is too large for email. "
+    "To receive it, see https://gcn.nasa.gov/quickstart"
+)
+
+
+def replace_long_values(data, max_length):
+    if isinstance(data, dict):
+        for key, value in data.items():
+            if isinstance(value, (str, bytes)) and len(value) > max_length:
+                data[key] = REPLACEMENT_TEXT
+            else:
+                replace_long_values(value, max_length)
+    elif isinstance(data, list):
+        for index, item in enumerate(data):
+            if isinstance(item, (str, bytes)) and len(item) > max_length:
+                data[index] = REPLACEMENT_TEXT
+            else:
+                replace_long_values(item, max_length)
```

Comment:
- Refactor duplicate code
- Deserialization of JSON can never produce bytes

```suggestion
    if isinstance(data, dict):
        iter = data.items()
    elif isinstance(data, list):
        iter = enumerate(data)
    else:
        return
    for key, value in iter:
        if isinstance(value, str) and len(value) > max_length:
            data[key] = REPLACEMENT_TEXT
        else:
            replace_long_values(value, max_length)
```